src/pages/Account/Tabs/ModulesTab.tsx

```diff
@@ -55,19 +80,78 @@ function ModulesContent({
   );
 }
 
-type ModulesTabProps = {
+function ModulesContent2({address}: {address: string}): JSX.Element {
+  const {data: registry} = useGetAccountResource(
+    address,
+    "0x1::code::PackageRegistry",
+  );
+
+  const [selectedModuleIndex, setSelectedModuleIndex] = React.useState<
+    number | null
+  >(null);
+
+  const packages: PackageMetadata[] =
+    registry === undefined ? [] : (registry.data as any).packages;
+  const modules = orderBy(
+    packages.flatMap((p) => p.modules),
+    "name",
+  );
+
+  if (modules.length === 0) {
+    return <EmptyTabContent />;
+  }
+
+  return (
+    <Grid container spacing={2}>
+      <Grid item md={4}>
+        {modules.map((module, i) => (
+          <Box key={module.name} onClick={() => setSelectedModuleIndex(i)}>
+            {module.name}
+          </Box>
+        ))}
+      </Grid>
+      <Grid item md={8}>
+        <ModuleContent
```

Comment:
https://user-images.githubusercontent.com/121921928/222226760-80c07389-4f4c-444f-a985-920cc039923f.mov

the screen recording shows that in the current explorer the first module will be open as default but not with this PR from netlify.